### DISCUSSION TITLE
how to create a custom dataset for train and test?

### DISCUSSION THREAD

**Discussion by kameronren** [deepjavalibrary/djl#2514] [Development]
For example human face train and test dataset。

**Comment by frankfliu** [comment]
You can take a look this example: https://github.com/deepjavalibrary/djl-demo/tree/master/footwear_classification#preparing-the-footwear-dataset

There is another example for creating a CSV dataset: https://github.com/deepjavalibrary/djl-demo/blob/master/malicious-url-detector/docs/dataset_creation.md

**Comment by kameronren** [comment]
thanks
